components/IngredientSelector/CategoryCard.tsx

```diff
@@ -1,18 +1,61 @@
 import { GenericCard } from "@/components/shared/molecules/GenericCard";
+import { BeefIcon } from "@/components/shared/atoms/icons/BeefIcon";
```

Comment:
Just curious, do we really need to create a .tsx file for each icon? We can't just link to the SVG file? (I'm not used to React at all so sorry if that's a dumb question, it just seems like a lot of boilerplate code)

Reply:
To have an ability to edit them

Reply:
Not sure to understand what do you mean by "edit them"?

You could have those icons in a SVG format and use them as follow. Might be easier for non react dev to just replace an SVG file and update the `src` than creating those tsx

```jsx
<img
  src="/static/illustration/eggs.svg"
  width="24"
/>
```